src/main/java/homework11/Main.java

```diff
@@ -0,0 +1,9 @@
+package homework11;
+
+public class Main {
+    public static void main(String[] args) {
+        FruitShop shop = new FruitShop();
+        shop.load("F:\\Slava\\homework10\\src\\main\\java\\homework11\\delivery");
```

Comment:
Please use a relative path.

Reply:
I dont know why, but if i us the relative path, fileReader cant read file ( throw FilenotFoundException) ... first i us the relative path

Reply:
It could not find the file. Please use google and find how to fix it. For example:
https://stackoverflow.com/questions/14209085/how-to-define-a-relative-path-in-java-windows